app/views/students/show.html.erb

```diff
@@ -5,10 +5,14 @@
 
 <p>
   <strong>Tutor:</strong>
-  <% if !@match.nil? %>
-    <%= Tutor.find(@match.tutor_id).name %>
-  <% else %>
-    No Tutor
+  <%= form_tag('set_tutor', method: 'put') do %>
```

Comment:
Rails doesn't automatically add the correct `authenticity_token` when using the `form_tag`. You can see the issue if you try setting a tutor on a student. You should see a `ActionController::InvalidAuthenticityToken` error. A couple ways to handle this would be to:

1. Use the `form_for` helper
2. Add a `hidden_field_tag` for the authenticity token
3. Add the `remote: true` option to the form.